bigquery/README.md

```diff
@@ -38,6 +38,10 @@ create it by running `gcloud auth application-default login`.
     ```
     docker-compose up -d elasticsearch
     ```
+  * If ES crashes due to OOM, you can increase [heap size](https://www.elastic.co/guide/en/elasticsearch/reference/current/heap-size.html):
```

Comment:
Can this be third bullet? This is much more rare than other two bullets.

Reply:
Done.